pegasus/forms/census.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './form'
+
+class Census2017 < Form
+  def self.normalize(data)
+    result = {}
+
+    result[:email_s] = required data[:email_s]
```

Comment:
to make sure this is a valid email address, you should add `email_address` (defined in lib/forms/pegasus_form_validation.rb) to the method chain here, e.g. `required email_address data[:email_s]`. 

This is equivalent to `required(email_address(data[:email_s]))`, which can help you remember that these chained methods are evaluated right to left.